Dictionaries/names.xml

```diff
@@ -226,7 +227,6 @@ This file is case sensitive.
   <name>Andre</name>
   <name>Andrea</name>
   <name>Andreas</name>
-  <name>AndreasB88</name>
```

Comment:
Query for this name couldn't fine anything link to it also it seems to be added back in 2010